src/download.jl

```diff
@@ -68,12 +69,6 @@ function determine_file(path, resp, hdrs)
                     )
 
         
-        # get the extension, if we are going to save it in encoded form.
-        # unlike a web-browser we don't automatically decompress
-        if header(resp, "Content-Encoding") == "gzip"
-            filename *= ".gz"
-        end
-        
```

Comment:
Auto decoding is added, so I think ".gz" is not needed anymore. And this change is needed to pass "test/download.jl".